### PR TITLE
WIP: Converting some MD to RST

### PR DESCRIPTION
**Pull request by yangchen1**
Converting some MD to RST

### CODE REVIEW

**Comment by it33** [comment]
Adding WIP to title

**Comment by esethna** [comment]
@yangchen1 can you please ask Lindy for help to review using the "view" button to look at the RST on GitHub?

**Comment by yangchen1** [comment]
thanks @it33 . and will do @esethna

File: source/help/settings/account-settings.md [hunk 1]
```diff
@@ -1,151 +0,0 @@
...
-Account Settings is accessible from the **Main Menu** by clicking the three dots at the top of the channels pane. From here, you can configure your profile settings, notification preferences, integrations, theme settings, and display options.
-
-## General
-Settings to configure name, username, nickname, email and profile picture.
-
-#### Full Name
-Full names appear in the Direct Messages member list and team management modal. By default, you will receive mention notifications when someone types your first name. Entering a full name is optional.
-
-#### Username
-Usernames are unique identifers appearing next to all posts. Pick something easy for teammates to recognize and recall. By default, you will receive mention notifications when someone types your username. In order to maintain message integrity, changing your username does not update @mentions in messages already posted.
```
Comment (lindy65) [inline source/help/settings/account-settings.md]: `identifers` > `identifiers`

File: source/help/settings/account-settings.md [hunk 2]
```diff
@@ -1,151 +0,0 @@
...
-Profile pictures appear next to all posts and in the top of the channels pane next to your name. All users have a generic profile picture when they sign up for an account. Edit your profile picture by clicking **Select** and then choosing a picture in either JPG or PNG format that’s at least 128px wide and 128px high. For best results, choose an image that is square with the subject of interest centered. If you edit your profile picture, all past posts will appear with the new picture.
-
-## Security
-Settings to configure your password, view access history, and view or logout of active sessions.
-
-#### Password
-You may change your password if you’ve logged in by email. If you are signing in using a single sign-on service, the password field is not editable, and you must access your SSO service’s account settings to update your password.
-
-#### Multi-Factor Authentication 
-When this option is available you can requires a phone-based passcode in addition your password for sign-in.
```
Comment (lindy65) [inline source/help/settings/account-settings.md]: Perhaps change this sentence to:
`When this option is available, a phone-based passcode is required in addition to your password for sign-in.`

File: source/help/settings/account-settings.md [hunk 3]
```diff
@@ -1,151 +0,0 @@
...
-Notification sounds fire for any activity that would trigger a desktop notification. Notification sounds are available on IE11, Edge, Safari and Chrome.
-
-##### Notification Duration
-
-Sets how long desktop notifications will remain on screen. If set to **Unlimited**, the user will have to manually clear each notification as it appears on screen. Desktop notifications in Safari can only stay on screen for a maximum of 5 seconds.
-
-#### Email Notifications
-Email notifications are sent for mentions and direct messages after you’ve been offline for more than 60 seconds or away from Mattermost for more than 5 minutes. Change the email where notifications are sent in **Account Settings > General > Email**.
-
-If your System Administrator has enabled **Email Batching**, you will have additional options under this settings to select how often email notifications will be sent. All notifications received over this time period are combined and sent in a single email.
```
Comment (lindy65) [inline source/help/settings/account-settings.md]: `settings` > `setting`

**Comment by lfbrock** [comment]
Closing this + YC and other PMs will resubmit in pieces, too many changes to merge.

**Pull request closed without merging**